docs/Ariba-connector.md

```diff
@@ -0,0 +1,25 @@
+# Ariba Connection
+
+Description
+-----------
+Use this connection to access data in Salesforce.
+
+Properties
+----------
+**Ariba Client ID (M)**: Ariba Client ID.
+
+**Ariba Client Secret (M)**: Ariba Client Secret.  
+
+**Ariba API Key (M)**: Ariba API Key.
+
+**API Endpoint (M)**: Base path of the Ariba API, for example, https://sandbox.api.sap.com. 
+
+**System Type**: Type of system the Ariba instance is running on: Production or Sandbox.  
+
+**Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.
+
+* **JSON**: Contents of the service account JSON file.
```

Comment:
I think you can delete JSON. Looks like a copy/paste error from a GCP plugin connection

Reply:
Removed